dtests/src/test/java/io/snappydata/hydra/connectionPool/hikariConnPool.inc

```diff
@@ -0,0 +1 @@
+io.snappydata.hydra.connectionPool.SnappyConnectionPoolPrms-useHikariConnPool = true;
```

Comment:
Use parameter with generic name and it can take names of connection pool to be used. e.g. useConnPool = ONEOF tomcat hikari FOENO